### ISSUE TITLE
luci-mod-status: Local Interface channels are not displayed on the graph

### ISSUE THREAD

**Issue by dvlom99** [openwrt/luci#6408]
## Steps to reproduce:
**Example: Replace the following lines and remove this**

1. go to: Status → Channel Analyzis -> radioX(5Ghz) or radioX(2.4Ghz)
2. wait for the scan to finish

## Actual behavior:
Local Interface channels not displayed on the graph:
![изображение](https://github.com/openwrt/luci/assets/60155978/c9fb9b76-52a6-42a0-aba1-5cc96e7f4311)
![изображение](https://github.com/openwrt/luci/assets/60155978/0eba84c0-c500-4525-af43-f5e6b815554d)

## Expected behavior:
Local Interface channels are correctly shown on the graph

## Additional Information:
OpenWrt version information from system `/etc/openwrt_release`

```
DISTRIB_ID='OpenWrt'
DISTRIB_RELEASE='22.03.5'
DISTRIB_REVISION='r20134-5f15225c1e'
DISTRIB_TARGET='ramips/mt7621'
DISTRIB_ARCH='mipsel_24kc'
DISTRIB_DESCRIPTION='OpenWrt 22.03.5 r20134-5f15225c1e'
DISTRIB_TAINTS=''
```

**Comment by jow-** [comment]
@Ansuel

**Comment by Ansuel** [comment]
@jow- i just notice that this is on 22.03 and this is fixed in master

Required change to be backported https://github.com/openwrt/luci/commit/307e385ba19fff36e335d048c45fc8f667129bd8

**Comment by jow-** [comment]
Fixed by 1c26abc138
